pkg/models/secrets_mount.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright © 2021 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package models
+
+type SecretsMountFormat struct {
+	Name string
+	Type string
+}
+
+var JSONMountFormat = &SecretsMountFormat{
+	Name: "json",
+	Type: "json",
+}
+var EnvMountFormat = &SecretsMountFormat{
+	Name: "env",
+	Type: "env",
+}
+var TemplateMountFormat = &SecretsMountFormat{
+	Name: "template",
+	Type: "template",
+}
+var DotNETJSONMountFormat = &SecretsMountFormat{
+	Name: "dotnet-json",
+	Type: "dotnet-json",
+}
+var SecretsMountFormatList = []*SecretsMountFormat{
+	EnvMountFormat,
+	JSONMountFormat,
+	DotNETJSONMountFormat,
+	TemplateMountFormat,
+}
```

Comment:
Can these all be declared `const`?

Reply:
It doesn't look like Go supports const for structs so I've refactored this to use strings instead.